orchestrator-salt/src/main/resources/salt/salt/gateway/scripts/generate-repo-for-os-from-vdf.sh.j2

```diff
@@ -18,7 +18,16 @@ fi
 
 echo "Using vdf url $VDF_URL"
 
-wget $VDF_URL --output-document=vdf.xml
+{% if pillar['proxy'] is defined %}
```

Comment:
I would use the same condition as here:
https://github.com/hortonworks/cloudbreak/blob/faf15fe172bcdea41a63df0beec66b9c58596896/orchestrator-salt/src/main/resources/salt/salt/pkg-mgr-proxy/init.sls#L1

Reply:
Yes initially this wasn't working well, because pillar['proxy'] is defined condition itself returns None, and it get's evaluated as true.
Maybe the one you suggested is longer but safer, so I'll add that.